packages/experimental/RadioGroup/src/RadioGroup/RadioGroup.styling.ts

```diff
@@ -14,12 +16,13 @@ export const stylingSettings: UseStylingOptions<RadioGroupProps, RadioGroupSlotP
       },
     })),
     label: buildProps(
-      (tokens: RadioGroupTokens, theme: Theme) => ({
+      (tokens: RadioGroupTokens) => ({
+        variant: tokens.variant,
         style: {
-          ...fontStyles.from(tokens, theme),
```

Comment:
Don't we want the font tokens to apply to the label?

Reply:
I set a variant instead versus defining all of the font tokens separately. Should I still include it?

Reply:
Someone could pass in a variant as a base and also override parts of the variant if they'd like to. Probably not likely to happen but we still support it, so yeah, we should include them.

Reply:
Makes sense, updated!